apps/web/src/components/HyloEditor/HyloEditor.js

```diff
@@ -157,7 +195,24 @@ const HyloEditor = React.forwardRef(({
 
     if (groupIds) editor.extensionStorage.mention.groupIds = groupIds
 
-    editor.setEditable(!readOnly)
+    // Wrap in setTimeout to ensure editor is ready
```

Comment:
why did we need this? more comments please. or comments in the PR at least

Reply:
Moved into useEffect (207) to respond to prop changes, its in a setTimeout to make sure we wait until the editor is fully initialized, put into a try/catch to get better error reporting... Trying to diagnose weird loading issues and get it all to look nice and work well.

Reply:
A setTimeout in this area worries me, but I do know that getting the ready state with TipTap can sometimes be tricky...